Fixed PHP Warning:  session_start():

baikaresandip opened the pull request:
I was getting warning while I am working on frontwnd and my debug.log was full of the error: 
`[21-Sep-2022 14:05:26 UTC] PHP Warning:  session_start(): Session cannot be started after headers have already been sent in /var/www/personalwp/public_html/wp-content/plugins/moowoodle/moowoodle.php on line 60` 
Fixed this warning.

baikaresandip commented:
Hi @kundandualcube  and  @purnendudash Who is mentaining this repo? Can anyone merge this PR?

purnendudash commented:
We need to check the code before the commit rather than a patch.

baikaresandip commented:
Yes sure. Please

kndnsow commented:
@baikaresandip where you getting the error
please share some references, that help to check the actual issue

baikaresandip commented:
HI @kundandualcube 
If you see the my commit there I have added error log message and fixed it. You can see it here:
https://github.com/dualcube/moowoodle/pull/50/commits/27936e4be25645d5c582834ad49bd522e97c174e

moumitahalder commented:
@baikaresandip, I understand that.

But first we need to recreate this on our end, only then we can apply the fix and verify. 

Please do share the steps on how can we create the issue on our end.

moumitahalder commented:
@baikaresandip any update?
